src/data/provinces.rs

```diff
@@ -265,7 +266,8 @@ pub struct Adjacency {
     from: ProvId,
     to: ProvId,
     // TODO: check type is "sea" or "river_large"
-    kind: Token, // sea or river_large
+    kind: Token,
+    // sea or river_large
```

Comment:
Hmm putting this comment on the next line makes it read wrong. It's a comment about `kind`